test/e2e/framework/test_context.go

```diff
@@ -26,12 +26,13 @@ import (
 )
 
 type TestContextType struct {
-	KubeConfig            string
-	KubeContext           string
-	KubeAPIContentType    string
-	KubeVolumeDir         string
-	CertDir               string
-	Host                  string
+	KubeConfig         string
+	KubeContext        string
+	KubeAPIContentType string
+	KubeVolumeDir      string
+	CertDir            string
+	Host               string
+	// TODO: Deprecating this over time... instead just use gobindata_util.go , see #23987.
 	RepoRoot              string
```

Comment:
A few somewhat iffy / weird tests.  I didn't want to touch them because they do weird stuff like embedding shell scripts and so on.  It would be wacky to embed shell scripts into bindata, and better probably to delete those tests over time